build-system/tasks/visual-diff/browser.js

```diff
@@ -0,0 +1,196 @@
+'use strict';
+
+const argv = require('minimist')(process.argv.slice(2));
+const fs = require('fs');
+const path = require('path');
+const puppeteer = require('puppeteer');
+const PuppeteerExtraPluginUserPreferences = require('puppeteer-extra-plugin-user-preferences');
+const {addExtra} = require('puppeteer-extra');
+const {cyan, yellow} = require('kleur/colors');
+const {HOST} = require('./consts');
+const {log} = require('./log');
+
+// REPEATING TODO(@ampproject/wg-infra): Update this whenever the Percy backend
+// starts using a new version of Chrome to render DOM snapshots.
+//
+// Steps:
+// 1. Open a recent Percy build, and click the “ⓘ” icon
+// 2. Note the Chrome major version at the bottom
+// 3. Look up the full version at https://en.wikipedia.org/wiki/Google_Chrome_version_history
+// 4. Open https://omahaproxy.appspot.com in a browser
+// 5. Go to "Tools" -> "Version information"
+// 6. Paste the full version (add ".0" at the end) in the "Version" field and click "Lookup"
+// 7. Copy the value next to "Branch Base Position" and update the line below
+const PUPPETEER_CHROMIUM_REVISION = '870763'; // 91.0.4472.0
```

Comment:
Not directly related to this PR: I'm concerned that at some point, pinning to v91 will prevent us from detecting bugs on newer versions of Chrome. Is there any value in running the tests on the latest version even if they are eventually rendered on v91 by Percy?

Reply:
Not really, this is just about the rendering. Percy does update its engine a few times a year, so we can always keep track of that